.github/workflows/install_firedrake_test.yml

```diff
@@ -0,0 +1,86 @@
+name: 'Install Firedrake and Mesh Adaptation software'
+
+on:
+  # Trigger when this or any file in the install directory changes
+  pull_request:
+    paths:
+      - 'install/**'
+      - '.github/workflows/install_firedrake_test.yml'
+
+  # Trigger the workflow every Saturday at 1AM
+  schedule:
+    - cron: '0 1 * * 6'
+
+  # Manually trigger the workflow
+  workflow_dispatch:
+
+# Stop the workflow if a new run is requested
+concurrency:
+  group: 'install_firedrake_test-${{ github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  install_firedrake:
+    name: 'Install Firedrake'
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:24.04
+      options: --user root
+
+    steps:
+      - name: 'Install required packages'
+        run: |
+          apt-get update
+          apt-get -y dist-upgrade
+          apt-get install -y --no-install-recommends tzdata curl vim docker.io \
+          openssh-client build-essential autoconf automake cmake gfortran git \
+          libopenblas-serial-dev libtool python3-dev python3-pip python3-tk \
+          python3-venv python3-requests zlib1g-dev libboost-dev sudo gmsh bison flex \
+          ninja-build libocct-ocaf-dev libocct-data-exchange-dev swig graphviz \
+          libcurl4-openssl-dev libxml2-dev
+
+      - name: 'Download necessary files'
+        run: |
+          curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/fix-mpi-install/install/Makefile
+          curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/fix-mpi-install/install/petsc_configure_options.txt
+
+      - name: 'Install Firedrake'
+        run: |
+          yes | make install_no_chk
```

Comment:
Is it necessary to use the `yes |` here?

Reply:
I think the CI is noninteractive by default so I think it's unnecessary, yup :) changed in https://github.com/mesh-adaptation/docs/pull/77/commits/d122f9b364c719bfb4aa543ace8062c31183e992. And btw in your PR I'd probably change this to `make install` (I had to use `install_no_chk` here because the `install` target was broken)

Reply:
No need to change it in my PR because this will update that branch when the merge goes through.

Reply:
I misunderstood but this makes sense - thanks!

Reply:
The `yes` was necessary to pass the `check_env`. Added it back in https://github.com/mesh-adaptation/docs/pull/77/commits/f1e84a2650189516643ab6adca38216d6cc21c92